Library/ViewModels/PledgeViewModel.swift

```diff
@@ -740,7 +740,7 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
         isApplePay: isApplePay
       )
 
-      return (data.project, baseReward, checkoutPropsData)
+      return (data.project, baseReward, checkoutPropsData, .newPledge)
```

Comment:
I don't think we should be hard-coding `.newPledge` here. After re-factoring I think we can remove this parameter.